geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureStore.scala

```diff
@@ -50,6 +52,8 @@ class AccumuloFeatureStore(val dataStore: AccumuloDataStore, val featureName: Na
         val feature = iter.next()
         val newFeature = fw.next()
 
+        minMaxVisitorO.foreach ( _.visit(feature) )
```

Comment:
Need to think a bit. I discussed it with other guys. The problem here is collection.features is an iterator and is empty when traversing for the second time.